src/mapping/mapping_helpers.py

```diff
@@ -2,47 +2,116 @@
 import pandas as pd
 import logging
 
-from src.utils.wrappers import time_logger_wrap, exception_wrap
-
 MappingLogger = logging.getLogger(__name__)
 
 
-def validate_ultfoc_df(df: pd.DataFrame) -> pd.DataFrame:
+def col_validation_checks(
+    df: pd.DataFrame,
```

Comment:
can you call this mapper_df, if it's going to be specific to mappers? I first thought it was the main df we were joining to.

Reply:
Ah, yes - that's much clearer